LICENSE

```diff
@@ -1,6 +1,6 @@
 MIT License
 
-Copyright (c) 2016   
+Crown Copyright (c) 2016-2017
```

Comment:
Hi,
Thanks for your contribution!
It however needs to be:
`Copyright (c) 2016-2017 Crown`
as per https://opensource.org/licenses/MIT

Thanks!
Chris